src/libexpr/flake/lockfile.hh

```diff
@@ -2,7 +2,7 @@
 
 #include "flakeref.hh"
 
-#include <nlohmann/json_fwd.hpp>
+#include <nlohmann/json.hpp>
```

Comment:
No no, we should not include json.hpp in header files! IIRC it adds a huge amount of compilation time to every compilation unit.